packages/itwin/infrastructure-iot/package.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "@itwin/infrastructure-iot",
+  "version": "0.1.0",
+  "description": "This package package provides pre-built UI components and data services to integrate Infrastructure IoT real-time sensor data into an iTwin application",
+  "license": "MIT",
+  "keywords": [
+    "Bentley",
+    "Infrastructure IoT",
+    "iTwin IoT Data",
+    "Digital Twin",
+    "iTwin.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/itwin/viewer-components-react/tree/master/packages/itwin/infrastructure-iot"
+  },
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/cjs/infrastructure-iot.js",
+  "module": "lib/esm/infrastructure-iot.js",
+  "typings": "lib/cjs/infrastructure-iot",
+  "scripts": {
+    "build": "npm run -s dual-build && npm run -s copy:assets",
+    "dual-build": "npm run -s build:cjs && npm run -s build:esm",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
+    "copy:assets": "npm run -s copy:public && npm run -s copy:cjs && npm run -s copy:esm",
+    "copy:public": "cpx \"./public/**/*\" ./lib/public",
+    "copy:cjs": "cpx \"./src/**/*.scss\" ./lib/cjs",
+    "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
+    "cover": "",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint:fix": "npm run lint -- --fix",
+    "pseudolocalize": "",
+    "test": "",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "rebuild": "npm run clean && npm run build"
+  },
+  "dependencies": {
+    "@itwin/itwinui-react": "^1.34.2",
+    "axios": "^0.26.1",
+    "class-transformer": "^0.5.1",
+    "highcharts": "^10.0.0",
+    "highcharts-react-official": "^3.1.0",
+    "lodash": "^4.17.21",
+    "moment": "^2.29.1",
+    "numeral": "^2.0.6",
```

Comment:
do you guys really need numeral?

Reply:
Refactored code to not require this and removed this dependency